slam-libraries/viam-cartographer/src/io/file_handler.cc

```diff
@@ -80,12 +80,11 @@ std::vector<std::string> ListSortedFilesInDirectory(
     return file_paths;
 }
 
-int RemoveFile(std::string file_path) {
+void RemoveFile(std::string file_path) {
     if (remove(file_path.c_str()) != 0) {
-        LOG(INFO) << "Error removing file";
-        return 0;
+        LOG(ERROR) << "Error removing file";
     }
-    return 1;
+    return;
```

Comment:
As tess pointed out in their comment above, the success/failure is not being used anywhere so we can change it to return void. 

We could also add in checks but since most operations would want to continue even if it could remove a file the log here is enough (alternatively we could change the log to return an error if desirable)

Reply:
Agree that a bool would be better if we continue return a value. I agree that failing to remove a file shouldn't throw. Logging the error sounds good.